topics/ecology/tutorials/x-array-map-plot/tutorial.md

```diff
@@ -0,0 +1,903 @@
+
+> ### Agenda
+>
+> In this tutorial we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+>### {% icon comment %} Background
+>
+>According to [UN](https://www.un.org/en/climatechange/what-is-climate-change) , Climate is the long term shift in temperature and weather patterns which may be due to natural or artificial causes. To learn more about climate, refer this [tutorial](https://training.galaxyproject.org/training-material/topics/climate/tutorials/climate-101/tutorial.html) from the GTN. Due to the frequently changing nature of the weather patterns, the size of the collected data is huge.
+The climate data is mainly represented in these three categories : [NetCDF](https://en.wikipedia.org/wiki/NetCDF) (Network Common Data Form), [HDF](https://en.wikipedia.org/wiki/Hierarchical_Data_Format) (Hierarchical Data Format) , [GRIB](https://en.wikipedia.org/wiki/GRIB) (GRIdded Binary or General Regularly-distributed Information in Binary form).
+>
+>The NetCDF file format is basically used for storing multidimensional data which generally consists of variables such as temperature, precipitation, wind direction, etc. The variation of climate variables over a period of time is suitably plotted using this dataset. The entire earth is divided into both horizontal as well as vertical coordinates which makes plotting of the variables such as the ocean temperatures possible.
+>
+>The coordinate system, types of projections and colormaps are some of the very important considerations in achieving the most suitable visualization option.
+
+
+
+# Introduction
+
+The data used in this tutorial is [ECMWF Reanalysis 5 hourly data on single levels](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview). We are interested in the following variables: air temperature at 2 metres, latitude, longitude and time. Our main objective is to plot the global air temperature at 2 metres with respect to time. For this we will be using the netCDF xarray tool available in the Galaxy Europe (or your favourite Galaxy Instance) server.
+
+It will be a fun learning experience for anyone who loves visualization ! 
+
+# Get data
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial. Name it as per your choice. My suggestions : *ECMWF_Reanalysis*.
+>
+>   
+>    {% snippet faqs/galaxy/datasets_import_from_remote_files.md %}
+>
+> 2. Import the files from
+>    the remote files
+>    ```
+>    (`Upload Data` -> `Choose remote files` -> `ECMWF ERA5 Reanalysis` >   ->`2022` -> `05` -> `data` -> `air_temperature_at_2_metres.nc`)
+>    ```
+>  or  upload it [from Zenodo](https://doi.org/10.5281/zenodo.6621460).
+>  
+> 3. Check that the datatype of uploaded data is **netCDF**. 
+>
+>
+>    If it is not `netCDF` make sure to convert it using the Galaxy built-in format converters.
+>
+>    {% snippet faqs/galaxy/datasets_convert_datatype.md conversion="Convert h5 to netCDF" %}
```

Comment:
@annefou @Quickbeasts51429 sorry for the late review comment, but should this be a real file conversion, or just a changing of datatype? (I suspect the latter, in which case you can use the `datasets_change_datatype.md` faq instead)

Reply:
Change in the datatype.